build.gradle

```diff
@@ -21,8 +21,8 @@ apply plugin: 'com.android.library'
 apply plugin: 'maven'
 
 android {
-  compileSdkVersion 23
-  buildToolsVersion '23.0.2'
+  compileSdkVersion 26
```

Comment:
tried just using the values from the example.

Failure is a gradle issue anyway... maybe my sdk failed to install somehow?